README.md

```diff
@@ -1716,6 +1716,13 @@ Thanks goes to these wonderful people ([:hugs:](https://allcontributors.org/docs
                     <sub><b>Kaone Keboetseng</b></sub>
                 </a>
             </td>
+            <td align="center">
+                <a href="https://github.com/SakshiRastogi1302">
+                    <img src="https://avatars.githubusercontent.com/u/80481744?v=4" width="100px;" alt="Kaone Keboetseng"/>
```

Comment:
```suggestion
                    <img src="https://avatars.githubusercontent.com/u/80481744?v=4" width="100px;" alt="Sakshi Rastogi"/>
```

Reply:
I have changed the alt name from Kaone Keboetseng to Sakshi Rastogi.